index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Bowling Scorecard</title>
+  <link rel="shortcut icon" type="image/png" href="images/Bowling.png">
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <h1>Bowling Scorecard</h1>
+  <Button id="start-game">Start Game</Button>
+
+  <div class="bowling-img">
+    <img src="https://media.giphy.com/media/l0HlW6GBSjbyFDCW4/giphy.gif" width="800">
+  </div>
+
+  <div class="kingpin hide">
+    <img src="images/kingpin.gif" width="90%">
+  </div>
+  
+  <div class="scorecard hide">
+    <div class="roll">
+      <h3>Select number of knocked pins for next roll:</h3>
+    </div>
+
+    <div class="scorecard-title">
+      <div>Frame</div>
+      <div>Roll 1</div>
+      <div>Roll 2</div>
+      <div>Roll 3</div>
+      <div>Score</div>
+      <div>Result</div>
+    </div>
+
+    <div class="scorecard-body">
+    </div>
+  </div>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+  <!-- <script type="module" src="src/frame.js"></script>
+  <script type="module" src="src/bowling.js"></script> -->
+  <script type="module" src="src/interface.js"></script>
```

Comment:
Using modules - fancy ;)
Make sure to remove commented out code when you clean up your code before a PR.